Monasca-log-agent fails to start if there is not connection to keystone

Tatarinho opened the pull request:
The purpose of this commit is to provide mechanism for monasca-log-agent
to wait for keystone service for determined amount of times instead of
failing at first try.

witekest reviewed (changes requested):
It's a cherry-pick from master branch, right? Please include this information in the commit message and PR description.
In future please use `git cherry-pick` command so that the according information is included automatically.

Tatarinho committed:
Monasca-log-agent fails to start if there is not connection to keystone

The purpose of this commit is to provide mechanism for monasca-log-agent
to wait for keystone service for determined amount of times instead of
failing at first try.

(cherry picked from commit aa2249715cd64efa8dbcfc7c22f7304ef2a385a8)

matrixik reviewed (approved):
Thank you.
